### DISCUSSION TITLE
Groupme bridge

### DISCUSSION THREAD

**Discussion by minecraftchest1** [42wim/matterbridge#1726] [New bridges suggestions]
**Is your feature request related to a problem? Please describe.**
Creating a Groupme bridge

**Describe the solution you'd like**
I am currently looking at creating a Groupme bridge, and am looking for any help, or anyone working on one already that I might be able to help

**Describe alternatives you've considered**
None.

**Additional context**
This seems to be fairly simple to do using bots or the [rest api](https://dev.groupme.com/docs/v3). If anyone is interested in helping, please reply here. I will post my code when I get something put together.

**Comment by minecraftchest1** [comment]
Here is my repro. [https://github.com/minecraftchest1/matterbridge-groupme](https://github.com/minecraftchest1/matterbridge-groupme)

**Comment by 42wim** [comment]
~https://github.com/densestvoid/groupme could be interesting~
Nevermind, we can't include GPL3 licensed libraries because of matterbridges apache license

**Comment by minecraftchest1** [comment]
Do I need to change my license from GPL3 to something compatible with yours, or just add an exception to the license?

**Comment by 42wim** [comment]
IANAL but according to https://www.apache.org/licenses/GPL-compatibility.html
> However, GPLv3 software cannot be included in Apache projects

So your license needs to be MIT / BSD or apache 2.0
![image](https://user-images.githubusercontent.com/1810977/106648742-f54b9780-6590-11eb-8758-9ccbe195b772.png)

Feedback of more knowledgeable people is appreciated :)

**Comment by minecraftchest1** [comment]
I have swapped the license from GPL3 to apache2.
